pr_slim.obo

```diff
@@ -14757,106 +14814,6 @@ intersection_of: PR:000019886 ! fibroblast growth factor 23, signal peptide remo
 intersection_of: only_in_taxon NCBITaxon:10090 ! Mus musculus
 relationship: only_in_taxon NCBITaxon:10090 ! Mus musculus
 
-[Term]
-id: PR:000059717
-name: ubiquitin (RPS27A) (mouse)
-namespace: protein
-def: "A ubiquitin (RPS27A) in mouse. UniProtKB:P62983, 1-76." [PRO:DAN]
-comment: Category=organism-modification.
-synonym: "mRPS27A/Clv:ubiq" EXACT PRO-short-label [PRO:DAN]
-synonym: "PRO_0000396479" EXACT PRO-proteoform-ftid [PRO:DNx]
-synonym: "UniProtKB:P62983, 1-76" EXACT PRO-proteoform-std [PRO:DNx]
-is_a: PR:000035716 ! ubiquitin (RPS27A)
-is_a: PR:P62983 ! ubiquitin-40S ribosomal protein S27a (mouse)
-intersection_of: PR:000035716 ! ubiquitin (RPS27A)
-intersection_of: only_in_taxon NCBITaxon:10090 ! Mus musculus
-relationship: only_in_taxon NCBITaxon:10090 ! Mus musculus
-
-[Term]
-id: PR:000060216
```

Comment:
Can you figure out why this is removed? Is does this PRO id appear anywhere in the seed? If not, everything in this PR looks good to me and can be merged.

Reply:
Thanks for checking, @matentzn. This ID does not appear in seed.txt.